content/accessibility/checklist.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Accessibility checklist
+---
+
+For official guidance, always refer to the W3C documentation. You can find failure examples for each WCAG standard, and techniques for fixing them in [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.1&showtechniques=148&currentsidebar=%23col_customize#visual-presentation). The filters on the left hand side can help when looking for specific subjects, such as interaction or visual design, content creation, and more. 
+
+If you're designing a feature or experience in GitHub, use the following checklist to assess your design for accessibility before developer handoff:
+
+- [ ]  Did you include annotations for:
+    - [ ]  Headings
+    - [ ]  Focus/Tab order
+    - [ ]  Screen reader labels
+    - [ ]  Landmarks (if applicable)
+- [ ]  Does the text meet color contrast (4.5:1 or 3:1 ratio depending on size)?
+- [ ]  Do graphic objects meet color contrast (3:1 ratio)?
+- [ ]  With color removed, does the design still make sense?
+- [ ]  If the user increases the text size, is the page still usable?
+- [ ]  When the screen reduces in size (down to 320px wide), is everything usable without scrolling horizontally? (There are a few exceptions to this rule: check  [1.4.10 - Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) for details)
+
+When designing a new component or control that isn't made up of one single piece of [standard semantic markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element), check out the Accessibility Portal's [custom control design checklist](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-ux).
+
+## Visual considerations
+
+### Color
+<!-- TODO: insert photo of our colors with pass/fail-->
+<!-- TODO: the following link doesn't exist.
```

Comment:
@ichelsea Let's not assume the other colour document exists. We need a few set of rules here that designers should follow.

Reply:
That sounds good. I didn't want to repeat content that we'd have in the color doc but if that won't be ready for a while I can develop this out!